Sources/MapboxMaps/Gestures/GestureOptions.swift

```diff
@@ -60,5 +72,20 @@ public struct GestureOptions: Equatable {
     /// Defaults to `UIScrollView.DecelerationRate.normal.rawValue`
     public var panDecelerationFactor: CGFloat = UIScrollView.DecelerationRate.normal.rawValue
 
+    /// By default, gestures rotate and zoom around the center of the gesture. Set this property to rotate and zoom around a fixed point instead.
+    ///
+    /// Setting this property to any valid point sets ``GestureOptions/pinchPanEnabled`` to `false`.
+    public var focalPoint: CGPoint? {
+        didSet {
+            if pinchPanEnabled, focalPoint != nil {
+                Log.warning(
+                    forMessage: "Gesture options misconfiguration detected: a fixed focal point cannot be specified with pinch pan enabled. Pinch pan will be disabled.",
+                    category: "Gestures")
+
+                pinchPanEnabled = false
```

Comment:
This behaviour might be a bit confusing? Can we cache the focal point while the pinchPan is in progress and restore it when the pinchPan is finished?

Reply:
Resolved according to https://github.com/mapbox/mapbox-maps-ios/pull/1122#discussion_r812119694